plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -1427,6 +1427,55 @@ static TopNFunction sqlStandard(Function<String, String> quote)
         }
     }
 
+    public RemoteIdentifiers getRemoteIdentifiers(Connection connection)
+    {
+        return new RemoteIdentifiers()
```

Comment:
I liked `DatabaseMetaDataRemoteIdentifierSupplier` maybe we should have `DatabaseRemoteIdentifiers` or better  `JdbcRemoteIdentifiers`?

Reply:
I also liked it, but it was tightly coupled with sql.Connection